### PR TITLE
Separate router configs, add requestDelay option

### LINKED ISSUE #9: Router options
Router should admit the following options

- [x] `host`
- [x] `requestDelay`
- [ ] `defaultResponseHeaders` - Not sure about this one...

### PR DESCRIPTION
**Pull request by oskarcieslik**
This partially closes #9

### CODE REVIEW

**Comment by oskarcieslik** [comment]
There's a problem with this PR, @zzarcon can you take a look when you have a moment? I don't know why interceptors aren't disabled..

**Comment by oskarcieslik** [comment]
BOOOOM, FUCK U MUTABILITY

File: src/router.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 import { interceptors } from './interceptors';
 
-const defaultConfig = {
-  host: '',
+const routerDefaultConfig = {
   strategies: ['fetch', 'XMLHttpRequest']
 };
 
+const interceptorDefaultConfig = {
+  host: '',
```
Comment (zzarcon) [inline src/router.js]: Cool!

File: src/router.js [hunk 2]
```diff
@@ -1,16 +1,22 @@
 import _ from 'lodash';
 import { interceptors } from './interceptors';
 
-const defaultConfig = {
-  host: '',
+const routerDefaultConfig = {
   strategies: ['fetch', 'XMLHttpRequest']
 };
 
+const interceptorDefaultConfig = {
```
Comment (zzarcon) [inline src/router.js]: Good one!

**Pull request merged**